### DISCUSSION TITLE
Connecting to Strapi questions

### DISCUSSION THREAD

**Discussion by smetsc** [gatsbyjs/gatsby#36403] [Help] [answered]
Hi,

I'm following the guide "Connecting to Strapi" (https://support.gatsbyjs.com/hc/en-us/articles/360052324714-Connecting-to-Strapi) and I have some questions.

It says to have a Strapi instance ready, should this be local, or should this already be hosted somewhere?

Can the Gatsby project come from a simple starter template?

Thanks for your help and time!

With kind regards,
Casimir Smets

**Comment by TylerBarnes** [comment, marked as answer]
@smetsc you should be able to source from a local Strapi instance during development. When building your Gatsby site on a CI service (Gatsby Cloud) you will need to have a hosted Strapi instance as Gatsby wont be able to access your local Strapi instance in that case.
